### DISCUSSION TITLE
Allow filtering by Status conditional

### DISCUSSION THREAD

**Discussion by shifuma** [craftcms/cms#12124] [Ideas]
I'd like to have a separate element sources for current entries and expired entries. Is it possible to add Status to the list of available filters in the condition builder?

**Comment by brandonkelly** [comment]
We can’t add Status currently because it would conflict with the Status menu in the toolbar. We could look into hiding the Status menu if the source criteria already specifies a status, though.

**Reply by shifuma** [reply]
Makes sense. I think the relative dates in 4.3 will actually solve this issue any way (Expiry Date = before today)
